src/test/regress/expected/citus_local_tables_mx.out

```diff
@@ -724,5 +724,72 @@ $$);
  (localhost,57638,t,0)
 (2 rows)
 
+-- test adding foreign table to metadata with the guc
+SET citus.use_citus_managed_tables TO ON;
+CREATE TABLE foreign_table_test (id integer NOT NULL, data text);
+INSERT INTO foreign_table_test VALUES (1, 'text_test');
+CREATE EXTENSION postgres_fdw;
+CREATE SERVER foreign_server
+        FOREIGN DATA WRAPPER postgres_fdw
+        OPTIONS (host 'localhost', port :'master_port', dbname 'regression');
+CREATE USER MAPPING FOR CURRENT_USER
```

Comment:
Lets also show that `ALTER USER MAPPING` works fine:
```SQL

alter user MAPPING FOR onderkalaci SERVER foreign_server OPTIONS (SET user 'nonexistiniguser');
-- query the table and fails
alter user MAPPING FOR onderkalaci SERVER foreign_server OPTIONS (SET user 'postgres');

Reply:
added, thanks

Reply:
For enterprise, we should propagate the following two, but lets do that later:
```SQL
 GRANT USAGE ON FOREIGN SERVER foreign_server TO non_superuser;

CREATE USER MAPPING FOR CURRENT_USER
        SERVER foreign_server
        OPTIONS (user 'non_superuser');
```